### DISCUSSION TITLE
Python version

### DISCUSSION THREAD

**Discussion by nessuarez** [emartinez-dev/renfe-bot#10] [Q&A] [answered]
Which version of Python are you using?

**Comment by emartinez-dev** [comment]
I developed it using python3.11, but it should work on any >= python3.9 I think

**Comment by nessuarez** [comment]
I had some problems with python 3.12. That's why I was asking you about it

El mar, 9 ene 2024 a las 10:17, Enrique Martinez ***@***.***>)
escribió:

> I developed it using python3.11, but it should work on any >= python3.9 I
> think
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/emartinez-dev/renfe-bot/discussions/10#discussioncomment-8062508>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AALMIBQ5QEZLZM7EFWVXXEDYNUDLZAVCNFSM6AAAAABBSXPF3GVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANRSGUYDQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

*Néstor J. Suárez Alfonso*
[image: Facebook] <https://www.facebook.com/nestor.suarez.alfonso> [image:
Twitter] <http://twitter.com/nessuarez> [image: LinkedIn]
<http://www.linkedin.com/in/nessuarez>

--
Disclaimer:
This message and any attached files transmitted with it, is confidential,
especially as regards personal data. It is intended solely for the use of
the
individual or entity to whom it is addressed. If you are not the intended
recipient
and have received this information in error or have accessed it for any
reason,
please notify us of this fact by email reply and then destroy or delete the
message,
refraining from any reproduction, use, alteration, filing or communication
to third
parties of this message and attached files on penalty of incurring legal
responsibilities. The sender does not guarantee the integrity, the
accuracy, the
swift delivery or the security of this email transmission, and assumes no
responsibility for any possible damage incurred through data capture, virus
incorporation or any manipulation carried out by third parties.

**Reply by emartinez-dev** [reply, marked as answer]
I used Python 3.12.1 to run it right now and it is working, I am happy to help you if you could provide me more information about the problems

![image](https://github.com/emartinez-dev/renfe-bot/assets/73340871/7e5cbb21-824c-48d3-aeee-35fdf99eb852)

**Comment by nessuarez** [comment]
I just realised that was before your last commit.

Thanks for the clarification!

El mar, 9 ene 2024 a las 14:40, Enrique Martinez ***@***.***>)
escribió:

> I used Python 3.12.1 to run it right now and it is working, I am happy to
> help you if you could provide me more information about the problems
>
> image.png (view on web)
> <https://github.com/emartinez-dev/renfe-bot/assets/73340871/7e5cbb21-824c-48d3-aeee-35fdf99eb852>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/emartinez-dev/renfe-bot/discussions/10#discussioncomment-8066707>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AALMIBTBUTHLTDZP7A2RUULYNVCERAVCNFSM6AAAAABBSXPF3GVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANRWG4YDO>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

*Néstor J. Suárez Alfonso*
[image: Facebook] <https://www.facebook.com/nestor.suarez.alfonso> [image:
Twitter] <http://twitter.com/nessuarez> [image: LinkedIn]
<http://www.linkedin.com/in/nessuarez>

--
Disclaimer:
This message and any attached files transmitted with it, is confidential,
especially as regards personal data. It is intended solely for the use of
the
individual or entity to whom it is addressed. If you are not the intended
recipient
and have received this information in error or have accessed it for any
reason,
please notify us of this fact by email reply and then destroy or delete the
message,
refraining from any reproduction, use, alteration, filing or communication
to third
parties of this message and attached files on penalty of incurring legal
responsibilities. The sender does not guarantee the integrity, the
accuracy, the
swift delivery or the security of this email transmission, and assumes no
responsibility for any possible damage incurred through data capture, virus
incorporation or any manipulation carried out by third parties.

**Comment by mjrodri** [comment]
I am running 3.8 on pycharm at the moment. I think its older on VS Code but i don't really like VS Code as much so I am not as up to date on it.
